src/modules/database/database.ts

```diff
@@ -0,0 +1,41 @@
+import mongoose from 'mongoose';
+
+class Database {
+  private static instace: Database | null = null;
+
+  private mongodb = mongoose;
+
+  private uri = 'mongodb://127.0.0.1:27017/local';
+
+  private constructor() {
+    this.dropDatabase();
+    this.connect();
```

Comment:
This method returns a promise and you are not using `await`, that's what you wanted ?

Reply:
You are right. I removed all unnecessary promises.

Reply:
You are right. I improved the code now.